### ISSUE TITLE
Ability to create "Zettelkasten"-notetitle

### ISSUE THREAD

**Issue by jrennstich** [argenos/zotero-mdnotes#59]
First of all THANK YOU so much for providing this wonderful tool! I am now able to maintain a working ["Zettelkasten"](https://zettelkasten.de/posts/overview/) system with Zotero and Devonthink 3 without yet another tool for organizing notes. 

One feature I would really like to automate is to create a unique ID for each note based on the now common notation of 
`YYYMMMDDHHMM`. This would go into the `<prefix>` of the file export setup. Maybe this is already possible but I wasn't able to work out, how to set up a placeholder/variable in the `prefix` field. 

Again, thanks so much for this amazing Zotero-plugin!

**Comment by tarahmarie** [comment]
I found this issue by searching for the same thing; I would love this. It's the only barrier between a perfect markdown export to Zettlr, Obsidian, etc, for me. If you're able to point to the location this is generated, I am happy to provide an option or work on this.

**Comment by elviskahoro** [comment]
Have you tried setting MDNotes to use the citation key as the title? And then including the YYYYMMDDHHMM in BIBTEX's citation key format?

**Comment by elviskahoro** [comment]
1. Set "Use the item's citekey as title?" to true in MDNotes preferences.
2. In BibTex prefernces for Citation Key Format include this label:
[Date:format-date=%Y%m%d%H%M%S]
note: I still have seconds included but you can remove this.
3. Add the other information you want to the bibtex citation key format:
[Date:format-date=%Y%m%d%H%M%S]   - [auth:lower] - [other_tags]

4. List of labels you can use in your citation keys are located here:
https://retorque.re/zotero-better-bibtex/citing/

**Comment by elviskahoro** [comment]
@jrennstich @tarahmarie Can you let me know if it's working? And then we can probably close this.

**Comment by tarahmarie** [comment]
For lack of this feature I'd deleted MD Notes; let me try again this week and see if it works for me...will report back.

**Comment by argenos** [comment]
@elviskahoro thanks for the workaround! I do have plans to add a wider support of dates in the file names and on templates, but unfortunately I haven't had time to sit down and work on mdnotes this past months. Happy to keep the issue open until I can implement this natively.

**Comment by elviskahoro** [comment]
If you're going to leave it open, then this issue should be low priority. Betterbib supports so much configuration for the citation key, why do extra work. I'll try to work on some issues myself! Thanks again for mdnores
